src/apps/settings/src/app/views/Metrics/Metrics.js

```diff
@@ -211,7 +243,13 @@ const Body = ({
 
   // Calculate number of requests for each type
   // This is a map of req code -> number of reqs
-  const reqs = { other: 0 };
+  const reqs = {
+    other: 0,
+    200: 0,
+    301: 0,
+    404: 0,
+    403: 0,
+  };
   for (let i = 0; i < requestData.ResponseCodes.length; i++) {
```

Comment:
Same comment that this can be done more delcaritive.

i.e requestData.ResponseCodes.forEach()